user_manual/webinterface.rst

```diff
@@ -14,29 +14,29 @@ Web browser requirements
 For the best experience with the Nextcloud web interface, we recommend that you use the 
 latest and supported version of a browser from this list:
 
-.. No need to translate
-
-* Microsoft **Internet Explorer**
 
 .. No need to translate
 
-* Microsoft **Edge**
+* Google **Chrome**/Chromium
 
 .. No need to translate
 
 * Mozilla **Firefox**
 
 .. No need to translate
 
-* Google **Chrome**/Chromium
+* Apple **Safari**
 
 .. No need to translate
 
-* Apple **Safari**
+* Microsoft **Edge**
 
 .. note:: If you want to use Nextcloud Talk you need to run Mozilla **Firefox** 52+
    or Google **Chrome**/Chromium 49+ to have the full experience with video calls and 
    screensharing. Google **Chrome**/Chromium requires a additional plugin for screensharing.
+   
+.. note:: While we try our best to support it, we do **NOT** recommend the usage of microsoft Edge.
```

Comment:
are you sure? Microsoft switched to chromium as base for microsoft edge a while ago...

Reply:
Yeah, even with that I think we had reports iirc. 
But also, most of us are actually never testing with that browser, could be a safe call to not recommend it.

Reply:
cc @juliushaertl @danxuliu @ChristophWurst @PVince81 @nickvergessen

Reply:
Edge is supported and should work flawlessly (apart from a minor issue in Talk)
I still don't recommend using it as it means you run windows, but should be fine in general

If you still keep this as least fix the typo:
```suggestion
.. note:: While we try our best to support it, we do **NOT** recommend the usage of Microsoft Edge.
```

Reply:
> I still don't recommend using it as it means you run windows

It also runs on macos, ios android and partially on Linux, if I remeber correctly: https://docs.microsoft.com/en-us/deployedge/microsoft-edge-supported-operating-systems

Reply:
Thanks for ruining the joke :P

Reply:
Sorry 😅